app/src/components/tables/ValidatorTable/ValidatorTable.tsx

```diff
@@ -2,40 +2,78 @@ import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from '@emotion/styled';
 import { ColumnDef } from '@tanstack/react-table';
-import { colors, fontWeight } from 'src/styled-theme';
+import { colors, pxToRem } from 'src/styled-theme';
 import { ValidatorWeight } from 'casper-js-sdk';
+import {
+  getTotalEraValidators,
+  getValidatorsTableOptions,
+  setValidatorTableOptions,
+  updateValidatorPageNum,
+  useAppSelector,
+} from 'src/store';
+import { SelectOptions } from 'src/components/layout/Header/Partials';
 import { Table } from '../../base';
+import { NumberedPagination } from '../Pagination';
+
+const rowCountSelectOptions: SelectOptions[] | null = [
+  { value: '5', label: '5 rows' },
+  { value: '10', label: '10 rows' },
+  { value: '20', label: '20 rows' },
```

Comment:
we should add `rows` to translations :)

Reply:
@mgibson-casper I approved the PR but please add this one thing to translations and we're good!

Reply:
Oh yeah good call! I also updated the blocks table, which uses the same array for the selection/option.